### DISCUSSION TITLE
not doing good

### DISCUSSION THREAD

**Discussion by Lalbabu2001** [iNavFlight/inav#10449] [General]
i built my first quad and have very high hopes but what i feel is my bird is not doing good. i am not from engineering background but my curiocity drives me to inav. i will try my best to assist. 
here my blackbox log please review , are my filters and gains need to be altered?
[Uploading blackbox_log_2024-10-22_154739.TXT…]()

https://file.io/JasmTqOhm4a1

**Comment by sensei-hacker** [comment]
When you say "I feel my bird is not doing good", can you be about 100X times more specific?

"Not doing good" could describe hundreds of different problems, so we wouldn't know what you need help with.

**Comment by Lalbabu2001** [comment]
Yeah it's like not flying stable, shaking, I tried lowering p gain but no luck. Please check out my filters are okay or not , I have attached log file
